src/Schema/Blueprint.php

```diff
@@ -94,7 +95,16 @@ public function tinyIncrements($column)
         $this->markAsNotSupported('AUTO_INCREMENT');
     }
 
-    // region Spanner Specific Types
+    /**
+     * @inheritDoc
+     * @return SpannerColumnDefinition
+     */
+    public function uuid($column = 'uuid')
+    {
+        $definition = new SpannerColumnDefinition(['type' => 'uuid', 'name' => $column]);
```

Comment:
How about overriding `addColumn()` method for consistency?

Reply:
That would make it more consistent but less robust. `generate_uuid` should only be available when a column is defined as uuid. I've renamed `ColumnDefinition` to `UuidColumnDefinition` for clarity.